docs/_data/portland-2021-schedule.yaml

```diff
@@ -0,0 +1,73 @@
+writing_day:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:30'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:45'
+    title: Team up and start working
+  - time: '12:30'
+    title: Lunch Break
+  - time: '5:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:00'
+    title: Write the Docs Team - Introduction to Write the Docs
+  - time: '9:45'
+    title: "<b>Unconference Starts</b>"
+  - time: '9:45'
+    slug: writing-a-perfect-technical-tutorial-jessica-garson
+  - time: '10:30'
+    slug: shuffle-ball-change-sashay-your-way-to-clearer-api-documentation-rachael-stavchansky
+  - time: '11:15'
+    slug: a-guide-to-getting-started-in-open-source-abigail-mccarthy
+  - time: '12:00'
+    slug: beyond-metrics-using-maturity-models-to-develop-a-docs-strategy-sarah-r-rodlund
+  - time: '12:45'
+    title: Lunch (60 mins)
+  - time: '1:45'
+    title: Lightning Talks (30 mins)
+  - time: '2:15'
+    slug: the-secret-history-of-libraries-paris-buttfield-addison
```

Comment:
Same re: europe timing.

Reply:
And of course Paris is in Australia. :+1: